### PR TITLE
(fix) Re: Issue #624 (https://github.com/node-gh/gh/issues/624)

### PR DESCRIPTION
**Pull request by keaglin**
`gh re --new` is working, tests passed. Current behavior: if a name is supplied, new remote repo will be given that name. If a name isn't supplied, it'll take the name of the pwd. 

Also: I noted what Zev said about doing set-url but that seemed like a whole other thing (probably need `child_process` from node or a 3rd-party package).

### CODE REVIEW

File: src/cmds/repo.ts [hunk 1]
```diff
@@ -341,7 +342,10 @@ Repo.prototype.run = function(done) {
         })
     }
 
-    if (options.new && !options.label) {
+    if (options.new !== undefined && !options.label) {
+        if (!options.new.trim()) {
+            options.repo, (options.new = getCurrentDirName())
```
Comment (protoEvangelion) [inline src/cmds/repo.ts]: What is the purpose of line 347? I am not familiar with that syntax.

**Comment by protoEvangelion** [comment]
Thanks for redoing this so quickly :) One thing I was curious about is how this syntax works:

```javascript
options.repo, (options.new = getCurrentDirName())
```

Have never seen that before.

**Comment by keaglin** [comment]
That's a good question. I... now don't remember what I was accomplishing there. Let me look at the code a little later today and I'll get back to you on that one.

**Comment by protoEvangelion** [comment]
So all it does is assign the current folder name to `options.new`.

If that is what you meant to do then we can just have:

```javascript
options.new = getCurrentDirName()
```

**Comment by protoEvangelion** [comment]
If that's the case let me know and I will make the change and merge 🎉

**Comment by keaglin** [comment]
Yeah I'm not sure what happened. It may have been something I forgot to clean up from an earlier version of this solution. But if taking off that random options.repo still works and passes the tests, I'm all for changing it!

**Comment by protoEvangelion** [comment]
hmm it's passing on everything except node v11.11 😢 

```
 FAIL  __tests__/git.test.ts
  ● Test suite failed to run
    TypeError: Cannot assign to read only property 'Symbol(Symbol.toStringTag)' of object '#<process>'
      at exports.default (node_modules/jest-util/build/create_process_object.js:15:34)
```

I have never seen this error before. Can you take a look?

**Comment by keaglin** [comment]
Sure. Give me a moment

**Comment by keaglin** [comment]
I pulled down changes and rebuilt it but I can't seem to reproduce that. Everything is passing for me.

**Comment by keaglin** [comment]
Sorry to kind of "works on my machine" you there but I can't pinpoint what might be different.

**Comment by protoEvangelion** [comment]
Thanks for your quick work on this @keaglin! Much appreciated 💯

**Comment by keaglin** [comment]
Absolutely! Happy to help 😄

**Pull request merged**